build.gradle

```diff
@@ -38,11 +41,14 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-api"
     testImplementation "io.micronaut.test:micronaut-test-junit5"
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
+
+    // Sub-Project dependencies
+    compile project(':core')
```

Comment:
I'm not a huge fan of having a `core` module. This usually ends up having the whole application in it, without clear distinction between logical units and possibly a big dependency mess. Is it already possible to split the core into independent modules, like `rfid`, `gpio`, `shared` (or `common` for shared classes)?

Reply:
'core' is in this case more 'backend' - no dependencies to micronaut.
What do you suggest? Should we habe sub projects for all packages like gpio / rfid?

Reply:
At least I would consider to separate the mpd- from the hardware-stuff - but this can be done in a separate step, as well.

Reply:
ill open a separate pr for this